.gitignore

```diff
@@ -7,5 +7,8 @@ vendor
 .vscode
 todo
 local.mk
+*.local.md
 .herumi
+dist/
+.releaser.yml
```

Comment:
Do we need this line?

Reply:
yeap actually the build files will reside in dist and `.releaser.yml` will be auto generated for os specific command

Reply:
yes actully the binaries generated will reside in `dist` and `.releaser.yml` will be generated for OS specific env